examples/dynamo/llama2_flashinfer_rmsnorm.py

```diff
@@ -0,0 +1,241 @@
+from typing import Callable, Optional, Sequence, Union
+
+import flashinfer
+import torch
+import torch_tensorrt
+from torch.fx.passes.shape_prop import TensorMetadata
+from torch_tensorrt.dynamo.lowering.passes._aten_lowering_pass import (
+    _aten_lowering_pass,
+)
+from torch_tensorrt.dynamo.lowering.passes.pass_utils import (
+    clean_up_graph_after_modifications,
+)
+from transformers import LlamaConfig, LlamaForCausalLM
+
+
+@torch.library.custom_op("flashinfer::rmsnorm", mutates_args=())  # type: ignore[misc]
+def flashinfer_rmsnorm(
+    input: torch.Tensor, weight: torch.Tensor, eps: float = 1e-6
+) -> torch.Tensor:
+    return flashinfer.norm.rmsnorm(input, weight)
+
+
+@torch.library.register_fake("flashinfer::rmsnorm")
+def _(input: torch.Tensor, weight: torch.Tensor, b: float = 1e-6) -> torch.Tensor:
+    return input
+
+
+torch_tensorrt.dynamo.conversion.plugins.custom_op(
+    "flashinfer::rmsnorm", supports_dynamic_shapes=True
+)
+
```

Comment:
TODO: After merge, extend this example to include an aot_impl

Reply:
There needs to be a modification to the plugin_converter generation that figure out if you can use aot_impl